core/app/models/spree/order.rb

```diff
@@ -70,6 +70,11 @@ def generate_number(options = {})
     has_many :products, through: :variants
     has_many :variants, through: :line_items
     has_many :refunds, through: :payments
+    has_many :all_adjustments,
+      class_name: 'Spree::Adjustment',
```

Comment:
@JDutil I do not think this recommendation makes sense here, as we than have to use explicit parenthesis about `has_many` DSL call.

```
has_many(
  :all_adjustments,
  class_name: ....,
  ...
)
```

Will not read that nice. Having the options in the same line violates the maximum line length, so I think the current format is the trade-off to do.